src/shapes/fabricObject.class.ts

```diff
@@ -0,0 +1,28 @@
+import { fabric } from '../../HEADER';
+import { FabricObjectSVGExportMixin } from '../mixins/object.svg_export';
+import { FabricObjectAncestryMixin } from '../mixins/object_ancestry.mixin';
+import { FabricObjectObjectAnimationMixin } from '../mixins/object_animation.mixin';
+import { InteractiveFabricObject } from '../mixins/object_interactivity.mixin';
+import { FabricObjectObjectStackingMixin } from '../mixins/object_stacking.mixin';
+import { FabricObjectObjectStraighteningMixin } from '../mixins/object_straightening.mixin';
+import { applyMixins } from '../util/applyMixins';
+
+// TODO somehow we have to make a tree-shakeable import
+
+applyMixins(InteractiveFabricObject, [
+  FabricObjectAncestryMixin,
+  FabricObjectObjectStackingMixin,
+  FabricObjectObjectStraighteningMixin,
+  FabricObjectSVGExportMixin,
+  FabricObjectObjectAnimationMixin,
+]);
+
+// export interface InteractiveFabricObject
+//   extends FabricObjectAncestryMixin,
+//     FabricObjectObjectStackingMixin,
+//     FabricObjectObjectStraighteningMixin,
+//     FabricObjectSVGExportMixin,FabricObjectObjectAnimationMixin {}
```

Comment:
The link of the TS mixins states to do this but it won't let me since (I think) it needs to be in the same file declaring the class

Reply:
fixed